Description: Build with xml-resolver instead of com.sun.org.apache.xml:resolver
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/jaxb-ri/xjc/src/main/java/com/sun/tools/xjc/CatalogUtil.java
+++ b/jaxb-ri/xjc/src/main/java/com/sun/tools/xjc/CatalogUtil.java
@@ -40,8 +40,8 @@
 
 package com.sun.tools.xjc;
 
-import com.sun.org.apache.xml.internal.resolver.CatalogManager;
-import com.sun.org.apache.xml.internal.resolver.tools.CatalogResolver;
+import org.apache.xml.resolver.CatalogManager;
+import org.apache.xml.resolver.tools.CatalogResolver;
 import java.io.File;
 import java.io.IOException;
 import java.net.URI;
